Nef_3/include/CGAL/Nef_3/SNC_point_locator.h

```diff
@@ -328,8 +328,8 @@ class SNC_point_locator_by_spatial_subdivision :
       case is_vertex_: return make_object(v_res);
       case is_edge_: return make_object(e_res);
       case is_facet_: return make_object(f_res);
+      case is_none_ : return Object_handle();;
```

Comment:
You wrote double `;;` 
```c++
case is_none_ : return Object_handle();;
```

Also, I think something similar should be done in `locate()` which also uses the solution enum. It might be good to initialise to `_is_none` in locate.  It uses an If/else If , though so I expect no warning there.

Reply:
Can you take care of these fixes. Thanks.